README.md

```diff
@@ -1,42 +1,41 @@
-Security Vocabulary
-===================
+# Security Vocabulary
 
-This repository contains the [Digital Verification Community Group][DVCG]
+This repository contains the [Credentials Community Group][ccg]
 Security Vocabulary and a [npm package][security-context] that exports related
 contexts and constants.
 
-Security Vocabulary
--------------------
+## Security Vocabulary
 
-Security Vocabulary specification: https://w3c-dvcg.github.io/security-vocab/.
+Security Vocabulary specification: https://w3c-ccg.github.io/security-vocab/.
 
-Context Package
----------------
+## Context Package
 
 The repository contains [JSON-LD][] contexts for the Security Vocabulary.
 These are also packaged in a [npm package][security-context] for CommonJS and
-ES Modules.  To use with NPM and Node.js, use the following:
+ES Modules. To use with NPM and Node.js, use the following:
 
 ```
 npm install security-context
 ```
 
 The package exposes two values:
+
 - `contexts`: A Map from context URI to JSON-LD context.
 - `constants`: An Object of shorthand keys mapped to context URIs.
 
 ```js
-const {contexts, constants} = require('security-context');
+const { contexts, constants } = require("security-context");
```

Comment:
My expectation is that @dlongley @mattcollier @davidlehn and @gannan08 may have issues with the double quotes and the spacing. If you don't touch the code, it'll be easier to merge this.